crates/matrix-sdk/src/widget/matrix.rs

```diff
@@ -47,6 +47,11 @@ pub(crate) struct MatrixDriver {
     room: Room,
 }
 
+#[derive(serde::Deserialize)]
+struct RedactBody {
```

Comment:
nit: add a doc comment explaining what this is, how it's used, where it comes from (I suppose this is part of the widget protocol? is it an input or an output?)

Reply:
I think I actually prefer to make the parsing from RawValue to the radacts index explicitly.